### PR TITLE
Email Signup Submission Fixed

### PR DESCRIPTION
**Pull request by BHS-Harish**
## Issue that this pull request solves

 Closes: #878 

## Proposed changes

Integrated into backend and toast message added.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Screenshots

Screenshot 2024-08-07 214247

### CODE REVIEW

**Comment by BHS-Harish** [comment]
Hey @Kajol-Kumari 

Please review it and merge it

File: frontend/src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -47,14 +54,15 @@ export const Footer = (props) => {
   };
 
   //handling submit
-  const handleSubmit = (e) => {
+  const handleSubmit = async(e) => {
     e.preventDefault();
     //if isValid = true, form submission trigger
     const isValid = validation();
     if (isValid) {
-      setSubmited(true);
```
Comment (Kajol-Kumari) [inline frontend/src/components/Footer/Footer.jsx:55]: For every new user, there should be a welcome email sent to the subscribers

**Review by Kajol-Kumari — COMMENTED** [review]
I hope we gave unique subscribers email check on backend

**Comment by BHS-Harish** [comment]
> I hope we gave unique subscribers email check on backend

Yeah It not stored if email already exists.

File: frontend/src/components/Footer/Footer.jsx (hunk 1 above)
Comment (BHS-Harish) [reply frontend/src/components/Footer/Footer.jsx:55]: It's a frontend part... the email sent part should be done in backend.
Merge the request and i will add the email functionality in new issue.

**Review by Kajol-Kumari — APPROVED** [review]

**Pull request merged**